### DISCUSSION TITLE
Release for .NET 8?

### DISCUSSION THREAD

**Discussion by chris-nissen** [BlazorFluentUI/BlazorFluentUI#420] [General]
@vnbaaij Are there any plans for a release with the updates for .NET 8? 

My team's project uses BlazorFluentUI, we're working on upgrading to .NET 8, and we have an issue with the onmouseenter and onmouseleave event handlers.

**Comment by vnbaaij** [comment]
Hi,

I already upgraded the project to be vuilt with .NET 8 SDK. Never published the package aparently. Done that just now so v8.0.0 should be available on NuGet now.

**NOTE - IMPORTANT - WARNING**
I have not done ANY testing on this nor have I upgraded anything to use .NET 8 specific functionality. Use this at your own risk. 

The library is not really supported anymore and I HIGHLY recommend you to swith to the [Fluent UI Blazor library](https://github.com/microsoft/fluentui-blazor) (where I'm also the maintainer of).

**Reply by chris-nissen** [reply]
We do plan to switch soon. Thanks for publishing that!
